caffe-grid/pom.xml

```diff
@@ -18,11 +18,28 @@
     <maven.compiler.target>1.7</maven.compiler.target>
     <hadoop.version>2.5.0</hadoop.version>
     <scala.version>2.10.4</scala.version>
```

Comment:
Sure -thx for that snippet . Doing now.  btw what does  "CLA is valid!" indicate?